go/private/cgo.bzl

```diff
@@ -32,18 +31,28 @@ def _mangle(ctx, src):
     mangled_stem = ctx.attr.out_dir + "/" + src_stem.replace('/', '_')
     return mangled_stem, src_ext 
 
+def _c_filter_options(options, blacklist):
+  filtered = []
```

Comment:
Shorter but marginally less readable. Feel free to ignore:

```
return [opt for opt in options
        if not any([opt.startswith(prefix) for prefix in blacklist])]
```

Reply:
Done